src/JsonRpcProxy/send.js

```diff
@@ -0,0 +1,21 @@
+const { createAsyncMiddleware } = require('json-rpc-engine');
+const { providerFactory } = require('js-conflux-sdk');
+
+class Agent {
```

Comment:
Seems we can omit this Class here, direct create an provider below

Reply:
modified